src/UserManager.php

```diff
@@ -147,11 +149,13 @@ public function removeUsersFromDeletionList(array $users): void {
     $userIdClaim = $this->configHelper->getConfiguration('azure.user_id_claim');
     $userIdField = $this->configHelper->getConfiguration('drupal.user_id_field');
 
-    $this->logger->info($this->formatPlural(
-      count($users),
-      'Retaining one user.',
-      'Retaining @count users.'
-    ));
+    $this->logger->info(
+      'Retaining %count users',
```

Comment:
For consistency with info message in `collectUsersForDeletionList`:

```suggestion
      'Retaining %count user(s)',
```